labellab-client/src/components/navbar/project.js

```diff
@@ -83,11 +156,17 @@ ProjectNavbar.propTypes = {
   history: PropTypes.object,
   user: PropTypes.object,
   fetchUser: PropTypes.func,
-  title: PropTypes.string
+  title: PropTypes.string,
+  project: PropTypes.object,
 }
 
 const mapStateToProps = state => {
   return {
+    labels: state.labels.labels,
+    project: state.projects.currentProject,
+    imageName: state.projects.currentProject.images ?
+                state.projects.currentProject.images.map(image => image.imageName)
+                : null,
```

Comment:
```
const mapStateToProps = state => {
const imageName = state.projects.currentProject.images ?
                state.projects.currentProject.images.map(image => image.imageName)
                : null
  return {
    labels: state.labels.labels,
    project: state.projects.currentProject,
    imageName,
```
It can be imporved like this.

Reply:
sure I will make the changes thanks!!